build.sbt

```diff
@@ -115,7 +116,22 @@ lazy val server = (project in file("server")) enablePlugins(JavaAppPackaging) se
   ),
   Compile / PB.targets := Seq(
     scalapb.gen() -> (Compile / sourceManaged).value / "scalapb"
-  )
+  ),
+  daemonUser in Docker := "root",
+  daemonUserUid in Docker := None,
+  // insert setcap to before entrypoint
+  dockerCommands := {
+    val commandLen = dockerCommands.value.length
+    (dockerCommands.value.take(commandLen-2)
+      ++ Seq(Cmd(
+                "RUN",
+                "setcap", """cap_net_bind_service=+eip""",
+              "$(readlink -f /opt/docker/bin/delta-sharing-server)"),
+              Cmd("RUN", "mkdir /config", "&&", "touch",
```

Comment:
We don't need to create this since we have asked the user to mount it.

Reply:
Sorry, I thought I read on stackoverflow that the file needed to exist inside the container or else it will be created as a directory, but I just found out that it will only be created as a directory if it didn't exist on the host.